### PR TITLE
Add new keyword request handler for new website

### LINKED ISSUE #130: Unable to set custom facet fields (`facet.field`)
### Checked for duplicates

No - I haven't checked

### 🐛 Describe the bug

When I did the following query, it does not set the facet fields as expected.

https://staging.com/services/search/search?q=jupiter&rows=0&facet=on&facet.field=investigation_ref&facet.field=instrument_ref&facet.field=target_ref&wt=json&facet.limit=-1

```json
"facet_counts": {
"facet_queries": {},
"facet_fields": {
"facet_pds_model_version": [
"1,pds4",
430
],
"facet_agency": [
"1,nasa",
430
],
"facet_type": [
"1,collection",
360,
"1,bundle",
69,
"1,document",
1
],
"facet_target": [
"1,planet",
401,
"1,satellite",
192,
"1,other",
138,
"1,asteroid",
29,
"1,calibration",
11,
"1,comet",
3
```

### 🕵️ Expected behavior

I expected the facet fields to be set

### 📜 To Reproduce

https://staging.com/services/search/search?q=jupiter&rows=0&facet=on&facet.field=investigation_ref&facet.field=instrument_ref&facet.field=target_ref&wt=json&facet.limit=-1

### 🖥 Environment Info

_No response_

### 📚 Version of Software Used

_No response_

### 🩺 Test Data / Additional context

_No response_

### 🦄 Related requirements

🦄 #xyz

### ⚙️ Engineering Details

_No response_

### 🎉 Integration & Test

_No response_

### PR DESCRIPTION
**Pull request by jordanpadams**
## 🗒️ Summary
Add new keyword request handler for new website

## ⚙️ Test Data and/or Report
From staging website:
Screenshot 2024-09-27 at 5 46 45 PM

## ♻️ Related Issues

Resolves #130

### CODE REVIEW

File: test/smoke-tests.sh [hunk 1]
```diff
@@ -22,57 +22,53 @@ OUTDIR=$DATA_HOME/test1
...
+                                      -C $LEGACY_HARVEST_HOME/conf/search/defaults \
+                                      -o $OUTDIR --verbose 0
+
+check_status $? "[ERROR] Harvest Failure"
+echo "[SUCCESS] Harvest Successful"
+
+echo "[INFO] Check solr docs match expected"
+SOLR_EXPECTED=$PARENTDIR/src/test/resources/data/test1/expected/solr_doc_expected.xml
+# Cleanse output file of elements that are specific to where the code is run
+egrep -v package_id $OUTDIR/solr-docs/solr_doc_0.xml | egrep -v file_ref_location | egrep -v file_ref_url > solr_doc_actual.xml
```
Comment (nutjob4life) [inline test/smoke-tests.sh:47]: You can launch a single `egrep` process (and shorten the line length) with:

    egrep -v 'package_id|file_ref_location|file_ref_url' $OUTDIR/solr-docs/solr_doc_0.xml > solr_doc_actual.xml

File: test/smoke-tests.sh [hunk 2]
```diff
@@ -85,7 +81,7 @@ echo "[SUCCESS] Catalog Solr Successful"
 echo "[INFO] Check solr docs match expected"
 SOLR_EXPECTED=$PARENTDIR/src/test/resources/data/test2/expected/solr_doc_expected.xml
 # Cleanse output file of elements that are specific to where the code is run
-egrep -v package_id $OUTDIR/usa_nasa_pds_jnosru_0xxx__jnosruedr.solr.xml | egrep -v file_ref_location | egrep -v file_ref_url > solr_doc_actual.xml
+egrep -v package_id $OUTDIR/usa_nasa_pds_jnosru_0xxx__jnosruedr.solr.xml | egrep -v file_ref_location | egrep -v file_ref_url | egrep -v modification_date > solr_doc_actual.xml
```
Comment (nutjob4life) [inline test/smoke-tests.sh:84]: Similar to above, maybe try:

    egrep -v 'package_id|file_ref_location|file_ref_url|modification_date' $OUTDIR/usa_nasa_pds_jnosru_0xxx__jnosruedr.solr.xml > solr_doc_actual.xml

**Review by nutjob4life — CHANGES_REQUESTED** [review]
Just a couple readability (line length) and performance (process reduction) comments 😉

File: test/smoke-tests.sh (hunk 1 above)
Comment (jordanpadams) [reply test/smoke-tests.sh:47]: updated

File: test/smoke-tests.sh (hunk 2 above)
Comment (jordanpadams) [reply test/smoke-tests.sh:84]: updated

**Pull request merged**